### DISCUSSION TITLE
Css style not looks goods

### DISCUSSION THREAD

**Discussion by smartdatasoft** [mostafizurhimself/admintoolkit-html#182] [General]
In mac safari list and many things do not show correctly. l like  
![Screenshot 2023-07-26 at 10 29 07 AM](https://github.com/mostafizurhimself/admintoolkit-html/assets/941400/3c753df1-d29b-498f-b760-8af62b59ff20)

**Comment by mostafizurhimself** [comment]
@smartdatasoft which version of safari, do you use?

**Reply by smartdatasoft** [reply]
Version 14.0 (15610.1.28.1.9, 15610)

**Reply by hasib-devs** [reply]
In my Safari browser, it works fine. Maybe it cause by the older version of Safari.
